### PR TITLE
avtp_pipeline: Add possibility to build avtp_pipeline without gstream…

### PR DESCRIPTION
**Pull request by PawelModrzejewski**
…er interfaces

AVB_FEATURE_GSTREAMER=0 should be used when building

### CODE REVIEW

File: lib/avtp_pipeline/avtp_pipeline.mk [hunk 1]
```diff
@@ -20,4 +21,5 @@ build/Makefile:
 	cmake -DCMAKE_TOOLCHAIN_FILE=../platform/Linux/x86_i210_linux.cmake \
 	      -DAVB_FEATURE_ENDPOINT=$(AVB_FEATURE_ENDPOINT) \
 	      -DIGB_LAUNCHTIME_ENABLED=$(IGB_LAUNCHTIME_ENABLED) \
+              -DAVB_FEATURE_GSTREAMER=$(AVB_FEATURE_GSTREAMER) \
```
Comment (andrew-elder) [inline lib/avtp_pipeline/avtp_pipeline.mk]: tabbing weirdness?

**Comment by MarcinMiklas** [comment]
It looks good at the first sight. I'll try to test if it compiles with and without GSTREAMER

**Comment by andrew-elder** [comment]
Thanks for testing @MarcinMiklas. @PawelModrzejewski  - are you able to quickly fix that tabbing issue?

**Comment by andrew-elder** [comment]
Looks good. Merging.

**Pull request merged**